### PR TITLE
Ticket 2/POST route with json biz registration

### PR DESCRIPTION
**Pull request by seintun**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ func main() {
 	router := mux.NewRouter()
 
 	router.HandleFunc("/biz/validate", getBizYelp).Methods("POST")
+	router.HandleFunc("/biz/register", registerBiz).Methods("POST")
```
Comment (developerX) [inline main.go]: Is this a new registered route for registration? Creates a user?

File: registerBiz.handler.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	Phone     string `json:"phone"`
+	Address   struct {
+		Address1 string `json:"address1"`
+		Address2 string `json:"address2"`
+		City     string `json:"city"`
+		State    string `json:"state"`
+		ZipCode  string `json:"zipCode"`
+	} `json:"address"`
+	YelpURL       string `json:"yelpURL"`
+	Img           string `json:"img"`
```
Comment (developerX) [inline registerBiz.handler.go]: What is this image for? are we storing an image?

**Comment by developerX** [comment]
How can I test this to make sure it works?

File: main.go (hunk 1 above)
Comment (seintun) [reply main.go]: This is Business Partner Registration after they press Yelp validate.

File: registerBiz.handler.go (hunk 2 above)
Comment (seintun) [reply registerBiz.handler.go]: This is a dummy link to display business profile picture.

**Pull request merged**